resources/views/individuals/partials/communication-and-consultation-preferences.blade.php

```diff
@@ -1,23 +1,14 @@
-<h3>{{ __('Preferred contact method') }}</h3>
-
-{!! Str::markdown($individual->user->primary_contact_method) !!}
-
-@if ($individual->user->alternate_contact_point)
-    <h3>{{ __('Alternate contact method') }}</h3>
-
-    @switch($individual->user->preferred_contact_method)
-        @case('phone')
-            <h4>{{ __('Email') }}</h4>
-        @break
-
-        @case('email')
-            <h4>{{ __('Phone') }}</h4>
-        @break
-
-        @default
-            <h4>{{ __('Phone') }}</h4>
-    @endswitch
-    {!! Str::markdown($individual->user->alternate_contact_method) !!}
+@if ($individual->user->preferred_contact_person === 'me')
+    <h3>{{ __('Contact :name', ['name' => $individual->name]) }}</h3>
+@else
+    <h3>{{ __('Contact :name’s support person, :support_person_name', ['name' => $individual->name, 'support_person_name' => $individual->user->support_person_name]) }}
+    </h3>
+@endif
+@if ($individual->contact_email)
+    <x-contact-point type='email' :value="$individual->contact_email" :preferred="$individual->preferred_contact_method === 'email' && $individual->contact_phone" />
+@endif
+@if ($individual->contact_phone)
+    <x-contact-point type='phone' :value="$individual->contact_phone" :preferred="$individual->preferred_contact_method === 'phone' && $individual->contact_email" :vrs="$individual->contact_vrs" />
 @endif
```

Comment:
It seems that in this case you are only marking something as preferred if it is the preferred contact type and there are both e-mail and phone numbers provided. However, in other cases you just check the preferred contact. Should this be consistent throughout?